src/core/Microsoft.PowerToys.Settings.UI/Views/FancyZonesPage.xaml

```diff
@@ -285,8 +284,6 @@
                      Margin="{StaticResource SmallTopMargin}"
                      Text="{x:Bind Mode=TwoWay, Path=ViewModel.ExcludedApps}"
```

Comment:
I wonder if we should move a part of this string to a ( i ) tooltip. It's rather long anyways, I can imagine with languages like Dutch or German this becomes almost a paragraph.

Reply:
lets treat that as a different issue than the root of this.